demo/webpack.config.js

```diff
@@ -10,7 +10,7 @@ var webpackConfig = {
         exclude: /node_modules/,
         loader: 'babel',
         query: {
-          presets: ['es2015'],
+          presets: ['env'],
```

Comment:
what does `env` do?

Reply:
Ahh it's basically the shiniest babel. Nice

Reply:
Yeah, for right now we'll get all of ES6-8 with this. But the real value is that then we can start specifying minimum browser versions we want to support via [browserslist](https://github.com/ai/browserslist) and Babel will only transpile what it needs to for our target envs.